ratbag_emu/endpoint.py

```diff
@@ -0,0 +1,107 @@
+# SPDX-License-Identifier: MIT
+
+import logging
+import struct
+import typing
+
+from typing import List
+
+from hidtools.uhid import UHIDDevice
+
+if typing.TYPE_CHECKING:
+    from ratbag_emu.device import Device  # pragma: no cover
+
+
+class Endpoint(UHIDDevice):
```

Comment:
I'd rather have the full `hidtools.uhid.UHIDDevice` instead of the `from ... import ...` above

Reply:
Yes. I agree.